lib/internal/repl.js

```diff
@@ -134,7 +152,7 @@ function setupHistory(repl, historyPath, ready) {
       return;
     }
     writing = true;
-    const historyData = JSON.stringify(repl.history, null, 2);
+    const historyData = repl.history.join(os.EOL);
```

Comment:
What if someone wants to share history files across platforms, maybe on a network drive? I think we'd be better off just using `\n` here.